nomad/structs/structs.go

```diff
@@ -7739,6 +7725,10 @@ func (t *Template) Copy() *Template {
 		nt.Wait = t.Wait.Copy()
 	}
 
+	if t.ChangeScript != nil {
+		nt.ChangeScript = t.ChangeScript.Copy()
+	}
```

Comment:
We nil check the receiver in the `Copy` and return nil, so we can safely leave it out here.